source/org.tensorflow/tensorflow-lite-support-api/Additions/Xamarin.TensorFlow.Lite.Support.Common.TensorProcessor.cs

```diff
@@ -0,0 +1,46 @@
+#nullable restore
+using System;
+using System.Collections.Generic;
+using Android.Runtime;
+using Java.Interop;
+
+namespace Xamarin.TensorFlow.Lite.Support.Common 
+{
+
+	// Metadata.xml XPath class reference: path="/api/package[@name='org.tensorflow.lite.support.common']/class[@name='TensorProcessor']"
+	// [global::Android.Runtime.Register ("org/tensorflow/lite/support/common/TensorProcessor", DoNotGenerateAcw=true)]
+	public partial class TensorProcessor // : global::Xamarin.TensorFlow.Lite.Support.Common.SequentialProcessor 
+    {
+		// [global::Android.Runtime.Register ("org/tensorflow/lite/support/common/TensorProcessor$Builder", DoNotGenerateAcw=true)]
+		public new partial class Builder // : global::Java.Lang.Object 
+        {
```

Comment:
Inconsistent whitespace: this appears to be spaces, should be tabs.

Reply:
fixed.
We added .editorconfig, not sure why VSCode did not pick it up